CHANGELOG.md

```diff
@@ -2,6 +2,30 @@
 
 This project follows [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v0.2.0 - 2024-12-09
+
+This is a backwards incompatible update. The following changes are needed:
+
+1. Replace all calls to `Alarmist.get_current_alarms/0` with `Alarmist.get_alarm_ids/0`. This is a hard deprecation.
```

Comment:
In v0.1.3, I see `Alarmist.current_alarms/0` and no `Alarmist.get_alarm_ids/0`

Reply:
`Alarmist.get_alarm_ids/0` isn't in v0.1.3. It's a new function. https://github.com/smartrent/alarmist/blob/main/lib/alarmist.ex#L103